tests/test_data_links.py

```diff
@@ -16,4 +16,15 @@ def test_collect_mappings_spec_v2(molecular_families_gnps):
     sut.collect_mappings_spec_v2(molecular_families_gnps)
     actual = sut.mapping_spec.shape
 
-    assert actual == (25935,3)
\ No newline at end of file
+    assert actual == (25935,3)
+
+
+def test_mappings_are_equal(spec_with_families, molecular_families_gnps):
+    sut = DataLinks()
+    sut.collect_mappings_spec(spec_with_families.values())
+    actual = sut.mapping_spec
+
+    sut.collect_mappings_spec_v2(molecular_families_gnps)
```

Comment:
It looks that `collect_mappings_spec`  and `collect_mappings_spec_v2` have the same functionality but different input types and implementations. In this case, I would suggest merging the functions to one function or changing them to two private functions and then creating a new public function to call them.